### DISCUSSION TITLE
CLI for building extensions

### DISCUSSION THREAD

**Discussion by eliashussary** [directus/directus#4790] [Feature Requests]
Instead of having to install `rollup` and a variety of plugins, it would be great if this came prepacked as a dev CLI from the `directus` team.

Something along the lines of `directus-dev build`

I'm happy to help contribute in support of this.

**Comment by rijkvanzanten** [comment]
I believe @Nitwel had a bunch of ideas regarding this too 🙂

**Reply by eliashussary** [reply]
I'm happy to help move this forward

**Comment by nickrum** [comment]
We have since added the Extensions SDK for this purpose 👍
